### PR TITLE
convert ifevals to ifdefs for build variable

### PR DESCRIPTION
**Pull request by jlsherrill**
Cherry-pick into:

* [ ] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

**Comment by jlsherrill** [comment]
For the most part this tries to 1to1 match the logic from ifevals to ifdefs.  as part of this though i will go through the pr call out things that are wrong and fix those.  

Things that are not incorrect but maybe just a little awkward (such as warning that a procedure is only for katello), i plant to leave alone.

**Comment by melcorr** [comment]
If this is ready, can you please rebase and I will review and see does everything look ok. Thank you !

File: guides/common/assembly_performing-additional-configuration-on-capsule-server.adoc [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
+ifndef::foreman-deb[]
 // Installing the Katello Agent
 include::modules/proc_installing-the-katello-agent.adoc[leveloffset=+1]
 endif::[]
 
 // Enabling OpenSCAP
 include::modules/proc_enabling-openscap.adoc[leveloffset=+1]
 
-ifeval::["{build}" != "foreman-deb"]
+ifndef::foreman-deb[]
```
Comment (jlsherrill) [inline guides/common/assembly_performing-additional-configuration-on-capsule-server.adoc:15]: gonna change both of these to ifdef::katello,satellite[]

File: guides/common/assembly_preparing-environment-for-capsule-installation.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@
 // System Requirements
 include::modules/ref_system-requirements.adoc[leveloffset=+1]
 
-ifeval::["{build}" != "foreman-deb"]
+ifndef::foreman-deb[]
```
Comment (jlsherrill) [inline guides/common/assembly_preparing-environment-for-capsule-installation.adoc:7]: changing this to  ifdef::katello,satellite[]

**Review by melcorr — APPROVED** [review]
I built the guides with these changes and all looks good. 
If this is ready, I can merge.

**Comment by jlsherrill** [comment]
@melcorr i think its ready!

**Pull request merged**